compiler/rustc_typeck/src/astconv/mod.rs

```diff
@@ -1911,6 +1945,14 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
                         has_err = true;
                         (ct.span, "const")
                     }
+                    hir::GenericArg::Infer(inf) => {
```

Comment:
t think we probably want to merge the ty + const output and always write "generic arguments"

Reply:
I think it is helpful as a user to know if generics are not allowed; or do you just mean for inference variables?